org.uqbar.project.wollok/src/org/uqbar/project/wollok/validation/WollokDslValidator.xtend

```diff
@@ -1372,6 +1380,14 @@ class WollokDslValidator extends AbstractConfigurableDslValidator {
 		}
 	}
 
+	@Check
+	@DefaultSeverity(WARN)
+	@CheckGroup(WollokCheckGroup.POTENTIAL_DESIGN_PROBLEM)
+	def describeWithEmptyDescription(WSuite s) {
```

Comment:
(nitpick) Lo mismo que el comentario de arrriba , variable = `suite`

Reply:
Sí, renombro a `it`